modules/ztp-image-based-upgrade-concept.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept_{context}"]
+= Managing the image-based upgrade at scale using the `ImageBasedGroupUpgrade` CR on the hub
+
+You can use a single resource, the `ImageBasedGroupUpgrade` custom resource (CR), to manage an imaged-based upgrade on a selected group of clusters through all stages.
+{cgu-operator-first} reconciles the `ImageBasedGroupUpgrade` CR and creates the underlying resources to complete the defined stage transitions, either in a manually controlled or a fully automated upgrade flow.
+
+The `ImageBasedGroupUpgrade` CR combines the `ImageBasedUpgrade` and `ClusterGroupUpgrade` APIs.
+For example, you can define the cluster selection and rollout strategy with the `ImageBasedGroupUpgrade` API in the same way as the `ClusterGroupUpgrade` API.
+The stage transitions are different from the `ImageBasedUpgrade` API.
+The `ImageBasedGroupUpgrade` API allows you to combine several stage transitions, also called actions, into one step that share one rollout strategy.
+
+[id="ztp-image-based-upgrade-supported-combinations_{context}"]
+== Supported action combinations
+
+Actions are the list of stage transitions that {cgu-operator} completes in the steps of an upgrade plan for the selected group of clusters.
+Each `action` entry in the `ImageBasedGroupUpgrade` CR is a separate step and a step contains one or several actions that share the same rollout strategy.
+You can achieve more control over the rollout strategy for each action by separating actions into steps.
+
+These actions can be combined differently in your upgrade plan and you can add subsequent steps later.
+Wait until the previous steps either complete or fail before adding a step to your plan.
+The first action of an added step for clusters that failed a previous steps must be either `Abort` or `Rollback`.
+
+[IMPORTANT]
+====
+You cannot remove actions or steps from an ongoing plan.
+====
+
+The following table shows example plans for different levels of control over the rollout strategy:
+
+.Example upgrade plans
+[cols=3*, width="100%", options="header"]
```

Comment:
Would it be better to rotate the table? i.e. 2 by 3 instead of 3 by 2?

Reply:
WDYT?
https://81685--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/ztp-image-based-upgrade.html#ztp-image-based-upgrade-supported-combinations_ztp-gitops